src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int base;
+    private int side;
+
+    public Rectangle(int base, int side) {
+        this.base = base;
+        this.side = side;
+    }

```

Comment:
This constructor is redundant, please remove it from all classes that `extends Figure` (constructors without `color`)